src/doc/en/installation/linux.rst

```diff
@@ -21,5 +21,5 @@ distribution, consider upgrading your distribution.
 Gentoo users might want to give a try to
 `sage-on-gentoo <https://github.com/cschwan/sage-on-gentoo>`_.
 
-The  :trac:`Trac wiki page Distribution <wiki/Distribution>` collects information
+The  :trac:`Issue wiki page Distribution <wiki/Distribution>` collects information
```

Comment:
Need a separate role for links to the new GH wiki

Reply:
True, though I'm just going to leave the doc folder alone for now.